function somethingWrongException() {
    return { msg: 'Algo de errado aconteceu, verifique os dados enviados e tente novamente!' };
}

export {
    somethingWrongException
}